webpack.config.externals.js

```diff
@@ -23,9 +23,8 @@ module.exports = {
                 loader: 'babel',
                 query: {
                     presets: [
-                        ["es2015", {"loose": true}],
-                        "react",
-                        "stage-2",
+                        "@babel/env",
```

Comment:
Can you add a .browserslistrc that mirrors what's in https://github.com/Khan/webapp/blob/master/.browserslistrc?  The `@babel/env` preset will look for that file and configure itself based on the entries in that file.

Reply:
Done!